toolchains/ci_workspaces/MODULE.bazel.tmpl

```diff
@@ -110,17 +89,23 @@ http_archive(
 # x86_64 python distribution. If using NVIDIA's version just point to the root of the package
 # for both versions here and do not use --config=pre-cxx11-abi
 
-#new_local_repository(
-#    name = "libtorch",
-#    path = "${TORCH_INSTALL_PATH}",
-#    build_file = "third_party/libtorch/BUILD"
-#)
+new_local_repository(
+    name = "libtorch_win",
+    path = "${TORCH_INSTALL_PATH}",
+    build_file = "third_party/libtorch/BUILD"
+)
 
-#new_local_repository(
-#    name = "libtorch_pre_cxx11_abi",
-#    path = "${TORCH_INSTALL_PATH}",
-#    build_file = "third_party/libtorch/BUILD"
-#
+new_local_repository(
```

Comment:
Keep in mind if we do this it becomes hard to build the cxx11 abi builds in CI. Consider just replacing the problematic one (libtorch_win) for now. Alternatively, we could replace both `libtorch_win`, and `libtorch_pre_cxx11_abi` but keep `libtorch` as `http_archive`, thats probably best of both worlds for CI

Reply:
Yes, this is a very good point, I have modified as suggested and also enabled release build to see ensure cxx11 abi build is successful in ci.